tests/parsers/test_sql_directory_parser.py

```diff
@@ -98,3 +98,10 @@ def test_parse_frontmatter(self):
             new_customers_table.operator.sql
             == "SELECT * FROM {customers_table} WHERE member_since > DATEADD(day, -7, '{{ execution_date }}')"
         )
+
+    def test_parse_creates_xcom(self):
+        with self.dag:
+            rendered_tasks = aql.render(dir_path + "/single_task_dag")
+        from tests.operators import utils as test_utils
+
+        test_utils.run_dag(self.dag)
```

Comment:
How are we confirming the table object was recorded in XCOM?
Are we missing any asserts in this test?

Reply:
@dimberman I would feel more comfortable if the test confirmed that one task depends on the other and had some more explicit checks. If someone inadvertently changed the files within `dir_path + "/single_task_dag"`, this test would likely continue working - but we would not be testing the original problem.